index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <main id="main">
+      <h2 id="title">Dr. Norman Borlaug</h2>
+      <div id="img-div">
+        <img id="image" />
```

Comment:
There are too many unnecessary divs, I will suggest semantic tags instead of divs i.e section.

Reply:
Great job 👍🏾 Here you could also direct them to an article or video for the student to have more depth about semantic tags. This could serve as a refresher for them.